src/lib.rs

```diff
@@ -136,20 +135,24 @@ impl manager::NotificationWatcher for ZWaveWatcher {
                 let controller = notification.get_controller();
                 let mut state = self.get_state();
                 if !state.controllers.contains(&controller) {
-                    println!("Found new controller: {:?}", controller);
+                    println!("Found new controller: {}", controller);
                     state.controllers.insert(controller);
                 }
             },
+            NotificationType::Type_NodeNew => {
+                let node = notification.get_node();
+                println!("NodeNew: {}", node);
```

Comment:
I wasn't sure how this might be useful, but I thought that we should at least print something so that we know when this happens. Ideally, I'd like to add a print for every single possible notification. Otherwise they just silently happen without us being aware of it.